src/main/java/org/cactoos/proc/BiProcOf.java

```diff
@@ -69,8 +82,8 @@ public BiProcOf(final BiFunc<? super X, ? super Y, ?> func) {
      */
     public BiProcOf(final Proc<? super X> prc) {
         this(
-            (x, y) -> {
-                prc.exec(x);
+            prc,
+            yvl -> {
```

Comment:
@andreoss please move this constructor above the one it calls

Reply:
@victornoel Done